### DISCUSSION TITLE
Kim karlı sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#172] [Zeka Sorusu] [answered]
A 100 ytl buldu, bununla B'den 100 ytl'lik mal aldı. B aynı parayla C'den 100 ytl'lik mal aldı.
C'nin A'ya 100 ytl borcu vardı, aynı parayla A'ya borcunu ödedi.
A paranın sahte olduğu fark ettiğinde iş işten geçmişti, çünkü C'ye senedi verdiğinden elinde senet yoktu ve parayı C'ye iade edemezdi. parayı şömineye attı.
Sahte paranın macerası sona erdiğinde kimin ne kadar karı ya da zararı oldu?

**Comment by FatihOnder81** [comment, marked as answer]
A bulduğu 100tl alışverişlerle c ye geçmiştir c nin a ya borcu vardır ve onu ödemiştir ama para sahtedir bu yüzden 100tl eksiye geçmiştir ama en başta b den 100tlik mal aldığı için o parayla herhangi bir kar zarar durumu yoktur b de aynı parayla c den mal aldığı için yine herhangi bir kar zarar durumu yoktur en son c den sahte parayla mal alındığı için zarara geçmiştir ancak 100tl lik borcunu sahte parayla ödediği için onunda kar zarar durumu yoktur  yani kimse kar veya zarar etmemiştir
